.gitignore

```diff
@@ -3,13 +3,16 @@ npm-debug.log
 yarn-error.log
 node_modules
 package-lock.json
+**/package-lock.json
+**/yarn.lock
```

Comment:
We can remove some redundant entries in the .gitignore file to make it cleaner and easier to maintain:
1. `package-lock.json` is already covered by the line above it. The `**/` prefix is unnecessary since the single entry will ignore this file in all directories.
2. `**/yarn.lock` is not needed if we're using Yarn v3, as the Yarn v3 section later in the file already handles ignoring the necessary Yarn-related files.

```suggestion
```